src/Kunstmaan/MediaBundle/Model/ImageUploadModel.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kunstmaan\MediaBundle\Model;
+
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class ImageUploadModel
```

Comment:
If we decide to keep this class I would make it final and add the `@internal` phpdoc to exclude it from the BC promise.